### ISSUE TITLE
For a release, devs need to manually create source-code tarballs for sc3-plugins

### ISSUE THREAD

**Issue by kristho91** [supercollider/sc3-plugins#134]
I'm getting following error when trying to build from the release tarball:

```
CMake Error at CMakeLists.txt:6 (message):
nova-simd submodule is missing: please run `git submodule init && git
submodule update' from the toplevel of your git working tree
```

This shouldn't be necessary, right? Or am I using the wrong tarball? (I am using this one: https://github.com/supercollider/sc3-plugins/archive/Version-3.8.0.tar.gz )

**Comment by vivid-synth** [comment]
@kristho91 yes, this seems wrong. In the meantime you can

```
$ git pull https://github.com/supercollider/sc3-plugins
$ cd sc3-plugins
$ git submodule update --init --recursive
```

**Comment by danstowell** [comment]
It's Github's fault. Github creates those tar.gz files automatically and github doesn't include submodules even though it should - see [this issue discussed on the main supercollider codebase](https://github.com/supercollider/supercollider/issues/1807). It is not the fault of the supercollider developers, and the supercollider developers have no way of removing those defective tarballs.

**Comment by kristho91** [comment]
@danstowell I know :) But could you maybe create a tarball that contains those submodules - like supercollider does on their own website?

**Comment by gusano** [comment]
@kristho91 Those tarballs (`Source code (zip)` and `Source code (tar.gz)`) are automagically created by github when creating a release and there's nothing we can do about it, as explained in [the issue Dan mentioned](https://github.com/supercollider/supercollider/issues/1807). All other tarballs are updated manually and contain the submodules.

**Comment by LFSaw** [comment]
we should

1. file an issue for github
2. mention it in the README?

**Comment by dvzrv** [comment]
Implemented with #186
